sdk/web-pubsub/web-pubsub-client/src/webPubSubClient.ts

```diff
@@ -521,6 +536,24 @@ export class WebPubSubClient {
     return new WebSocketClientFactory();
   }
 
+  private async _trySendSequenceAck(): Promise<void> {
+    if (!this._protocol.isReliableSubProtocol) {
+      return;
+    }
+    const [isUpdated, seqId] = this._sequenceId.tryGetSequenceId();
+    if (isUpdated) {
+      const message: SequenceAckMessage = {
+        kind: "sequenceAck",
+        sequenceId: seqId!,
```

Comment:
Should we check for seqId !== undefined as well? Maybe if(isUpdated && seqId)

Reply:
@joheredi  Make sense. Updated. Could you help to merge if no other concern